PcBdsPkg/Library/BootManagerSettingsDxeLibNull/BootManagerSettingsDxeLibNull.inf

```diff
@@ -0,0 +1,30 @@
+## @file
+# BootManagerSettingsLib - PcBds NULL BootManagerSettingsLib.
+#
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  EDK_RELEASE_VERSION             = 0x00020000
+  INF_VERSION                     = 1.27
+  BASE_NAME                       = DeviceBootManagerLibNull

```

Comment:
Shouldn't `BASE_NAME` be `BootManagerSettingsDxeLibNull`?

Reply:
Yes, it should.  Fixed.